docs/statistics.md

```diff
@@ -111,11 +119,31 @@ The time elapsed, in milliseconds, since the SRT socket has been created (after
 
 ### pktSentTotal
 
-The total number of sent data packets, including retransmitted packets. Available for sender.
+The total number of sent DATA packets, including retransmitted packets ([pktRetransTotal](#pktRetransTotal)). Available for sender.
+
+In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts sent packet filter control packets ([pktSndFilterExtraTotal](#pktSndFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
 
 ### pktRecvTotal
 
-The total number of received packets, including retransmitted packets. Available for receiver.
+The total number of received DATA packets, including retransmitted packets ([pktRcvRetransTotal](#pktRcvRetransTotal)). Available for receiver.
+
+In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts received packet filter control packets ([pktRcvFilterExtraTotal](#pktRcvFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
+
+### pktSentUniqueTotal 
+
+The number of unique DATA packets sent by the SRT sender. Available for sender. 
+
+This value contains only unique **original** DATA packets. Retransmitted DATA packets ([pktRetransTotal](#pktRetransTotal)) are not taken into account. In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](https://cac-word-edit.officeapps.live.com/we/API.md)), packet filter control packets ([pktSndFilterExtraTotal](https://cac-word-edit.officeapps.live.com/we/wordeditorframe.aspx?ui=en-GB&rs=en-US&wopisrc=https%3A%2F%2Fhai365.sharepoint.com%2Fsites%2FTeam-SRT%2F_vti_bin%2Fwopi.ashx%2Ffiles%2F421d054732ce45e08814ced46f033282&wdenableroaming=1&mscc=1&hid=53334D9F-F03D-B000-138D-338011C1234D&wdorigin=Other&jsapi=1&newsession=1&corrid=8ef94636-16cd-4a4f-ae0b-da2e3f2cacab&usid=8ef94636-16cd-4a4f-ae0b-da2e3f2cacab&instantedit=1&wopicomplete=1&wdredirectionreason=Unified_SingleFlush#pktSndFilterExtraTotal)) are not taken into account as well. 
```

Comment:
The link for `pktSndFilterExtraTotal` is somewhat excessive.

Per mechanism explanation, could be also added: This metric is counted for every case when a new (that is, not sent before) DATA packet has been sent over the network.